Atomic/util.py

```diff
@@ -474,3 +500,11 @@ def validate_manifest(spec, img_rootfs=None, img_tar=None, keywords=""):
     if keywords:
         cmd += ['-k',keywords]
     return subp(cmd)
+
+def get_signature_path(atomic_conf):
+    # If signature path is defined, get it; else return default
+    signature_path = get_atomic_config_item(['default-sigstore-path'], atomic_config=atomic_conf)
+    if signature_path is None:
+        signature_path = '/var/lib/atomic/containers'
```

Comment:
(See above; I don’t think there even should be a default location to be used for the whole internet, and then the issue of redundancy between the signing code and the argument parsing + help disappears.)